system-parachains/asset-hubs/asset-hub-polkadot/src/bridge_to_ethereum_config.rs

```diff
@@ -67,7 +67,7 @@ impl snowbridge_pallet_system_frontend::Config for Runtime {
 				Location,
 			>,
 		>,
-		EnsureRootWithSuccess<AccountId, RootLocation>,
+		EnsureRootWithSuccess<AccountId, RootLocation>, // TODO: Here or Parent?
```

Comment:
@claravanstaden Can you please confirm what do we want here, so we can  remote this TODO?

```suggestion
		EnsureRootWithSuccess<AccountId, RootLocation>,
```